addons/lunch/tests/test_supplier.py

```diff
@@ -84,64 +84,86 @@ def test_auto_email_send(self):
         with patch.object(fields.Datetime, 'now', return_value=self.monday_1pm) as _:
             with patch.object(fields.Date, 'today', return_value=self.monday_1pm.date()) as _:
                 with patch.object(fields.Date, 'context_today', return_value=self.monday_1pm.date()) as _:
```

Comment:
It's not a request change it's an info actually we can just do that if we don't want to indent to the sky and since you are remaking the tests maybe it could be better
from 3.1 (you can of course une backslaches to break the line)
```py
with A() as a, B() as b, C() as c: # the as is not mandatory
```
It's because in python it will call the context in order so it will for exemple do

```py

class mycontext():
    def __enter__(self):
        print('Starting1')
        return self

    def __exit__(self, *exc):
        print('Finishing1')
        return False

class mycontext2():
    def __enter__(self):
        print('Starting2')
        return self

    def __exit__(self, *exc):
        print('Finishing2')
        return False
 
with mycontext(), mycontext2():
    print("hello world")

>>> Starting1
Starting2
hello world
Finishing1
Finishing2
```

I think it's actually more readable

Reply:
up on this (not mandatory tho)